llvm/lib/Transforms/Utils/CodeExtractor.cpp

```diff
@@ -1594,6 +1594,10 @@ static void fixupDebugInfoPostExtraction(Function &OldFunc, Function &NewFunc,
         DPVsToDelete.push_back(&DPV);
         continue;
       }
+      if (DPV.isDbgAssign() && IsInvalidLocation(DPV.getAddress())) {
+        DPVsToDelete.push_back(&DPV);
+        continue;
+      }
```

Comment:
there's a comment a few lines up that needs deleting `// FIXME: support dbg.assign form of DPValues.`